infrastructure/cdn-in-a-box/enroller/enroller.go

```diff
@@ -318,10 +318,10 @@ func enrollDeliveryServiceServer(toSession *session, r io.Reader) error {
 		if len(servers.Response) == 0 {
 			return errors.New("no server with hostName " + sn)
 		}
-		if servers.Response[0].ID == nil {
-			return fmt.Errorf("Traffic Ops gave back a representation for server '%s' with null or undefined ID", sn)
+		if servers.Response[0].ID < 1 {
+			return fmt.Errorf("Traffic Ops gave back a representation for server '%s' with non-positive ID", sn)
```

Comment:
I think you should just get rid of this check instead of "fixing" it. It's not actually stated anywhere that all of our IDs will always be positive numbers, that's an implementation detail of an implementation detail. Clients needn't be expected to handle that.

Besides, the purpose of the check was to prevent a segmentation fault later during dereference, not to double-check that data received is valid.

Reply:
> I think you should just get rid of this check instead of "fixing" it.

Removed in 6268f0aaf